cuizinart/monitor.py

```diff
@@ -0,0 +1,99 @@
+import time
+import os
+from datetime import datetime
+from metadata_schema import *
+from watchdog.observers import Observer
+from watchdog.events import PatternMatchingEventHandler
+from extract_netcdf_header import *
+
+
+def parse_str_time(strs):
+    return datetime.strptime(strs, '%Y-%m-%d %H:%M:%S')
+
+
+def add_metadata(src_path):
+    product_name = src_path.split('/')[-2]
+    file_name = src_path.split('/')[-1]
+    product = Product.query.filter_by(key=product_name).first()
```

Comment:
Isn't the product name without the `forecast` prefix? I believe only the folder name contains the prefix, not the actual product name.

Reply:
The folder name is the product name. One product per folder.

Reply:
So you want all forecast product names to start with `forecast´? That seems a bit restrictive to me...

Reply:
No, only the folder name.

Reply:
Okay, great. So then I believe we need to change the `filter_by` key to search for product names without the prefix?

Reply:
Ok, I see what you mean. I was thinking about adding prefix as part of the product_name.